java/arcs/sdk/common/ReferenceId.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.sdk.common
```

Comment:
What's the logic of when to put something in arcs.sdk vs arcs.sdk.common?

Reply:
Common is useful to avoid situations where we'd have to tiptoe to avoid circular dependencies and other nastiness. I was considering putting it in arcs.sdk.storage, but felt that it would be better to not move it significantly from its analogous location in core.